turbo/jsonrpc/eth_receipts.go

```diff
@@ -73,7 +75,7 @@ func (api *APIImpl) GetLogs(ctx context.Context, crit filters.FilterCriteria) (t
 			return nil, err
 		}
 		if block == nil {
-			return nil, fmt.Errorf("block not found: %x", *crit.BlockHash)
+			return nil, errors.New(fmt.Sprintf("block not found: %x", *crit.BlockHash))
```

Comment:
what's the reason for all these `errors.New` wrapping changes? seems like the wrong thing to do - `fmt.Errorf` as it was before is what you'd normally use

Reply:
my main worry is this: https://github.com/erigontech/erigon/pull/12295/files#r1799861930
why does it make you change from `%w` to `%s`?
seems like it might be related to this option
```
    # Optimizes into `err.Error()` even if it is only equivalent for non-nil errors.
    # Default: false
    err-error: true
```

but does that mean the linter 100% checks all the usages up in the callers stack to see if there are any `errors.Is`?

Reply:
I guess we trust the linter for this.... 🤷‍♂️ 
double-checked and there are no `errors.Is` usages of callers of `GetBlockReceipts` so this is safe
but maybe we need to revisit this another time and do a test for the situation where there is actually a `errors.Is` or `errors.As` check in a parent caller